### PR TITLE
Add JDK 16 to build matrix

### PR DESCRIPTION
**Pull request by Egorand**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ jobs:
           - 13
           - 14
           - 15
+          - 16
```
Comment (ZacSweers) [inline .github/workflows/build.yml:24]:
```suggestion
          - 16
          - 17-ea
```

Go for gold?

**Review by ZacSweers — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (Egorand) [reply .github/workflows/build.yml:24]: Sure!

Comment (Egorand) [reply .github/workflows/build.yml:24]: Looks like Kotlin is not ready yet though

**Comment by ZacSweers** [comment]
That's weird, I'm using this fine in other kotlin projects.

**Pull request merged**